apps/web/src/services/dashboards/pages/DashboardsMainPage.vue

```diff
@@ -82,10 +82,9 @@ const state = reactive({
 
 const searchQueryHelper = new QueryHelper();
```

Comment:
I recall that we decided to avoid using QueryHelper in conjunction with reactive elements, or am I mistaken? It seems like we could reduce a lot of redundant code by using the useQueryTags hook instead.

Reply:
I missed it!! Thanks!!!

Reply:
😮